src/ucs/sys/sys.c

```diff
@@ -434,6 +434,21 @@ size_t ucs_get_page_size()
     return page_size;
 }
 
+int ucs_is_page_aligned(void *ptr)
+{
+    size_t page_size;
+
+    if (ptr) {
+        page_size = ucs_get_page_size();
+        if (ucs_likely(ucs_is_pow2(page_size))) {
```

Comment:
Check for `page_size > 0` fixes Coverity issue because it assumes the value maybe 0 due to `_n > 0` check inside `ucs_is_pow2`. Do you propose just to `return ptr % page_size` here?

If check for `ptr` and "faster" handling of pow2 are not needed, I see this function is meaningless. Should I remove it?